spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  describe '#initialize' do
+    it 'should be N business days later' do
+      t = DcTimePresenter.parse('2016-08-29 15:00')
```

Comment:
can we use `time` rather than `t`? I try to not use one-letter vars when possible because they can get confusing